Wikipedia/Localizations/qqq.lproj/Localizable.strings

```diff
@@ -474,3 +474,9 @@
 "status-saving-for-later" = "Indicates to the user that the article is being saved for later";
 "status-saved-for-later" = "Indicates to the user that the article has been saved for later";
 "announcements-dismiss" = "Button text indicating a user wants to dismiss an announcement\n{{Identical|No thanks}}";
+
+"places-search-top-articles" = "A search suggestion for top articles";
+"places-search-search-mapped-area" = "A button title that indicates the search will be redone in the mapped area";
```

Comment:
May want to consider describing this without using the word "mapped" just in case. 

Maybe something like...
> "Title for search button for finding results which are located within the currently visible area on the map"